### ISSUE TITLE
bin.log should take multiple args like console.log

### ISSUE THREAD

**Issue by SteveALee** [christianalfoni/webpack-bin#127]
`x=>bin.log('Prefix:', x)`

does not work. you have to concatenate the args

**Comment by bebraw** [comment]
Speaking of `bin.log`, it would be handy if there was a way to alias `console.log` to it (option somewhere?). That could be a good default even.

**Comment by christianalfoni** [comment]
Yeah, good points, both of you :) The output of multiple args would be the same as just running it multiple times though. It will be a UI structure hell to get it to actually log that stuff inline, considering nested objects, arrays etc.
